spec/features/user_cannot_access_admin_privileges_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe 'Access', type: :feature do
+  create_category
+  create_brand
+  create_items
+  create_user
+  create_rental
+
+    feature 'user' do
+      it 'cannot access admin screens' do
+
+      visit root_path
+      click_link "Login"
+
+      fill_in "Username", with: user_one.username
+      fill_in "Password", with: 'password'
+
+      click_button "Login"
+
+      visit new_admin_category_path
+
+      expect(page).to have_content("The page you were looking for doesn't exist.")
+    end
```

Comment:
`end` at 24, 4 is not aligned with `it 'cannot access admin screens' do` at 11, 6.